HW1/task4.py

```diff
@@ -0,0 +1,9 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе
+# они сделали S журавликов. Сколько журавликов сделал каждый
+# ребенок, если известно, что Петя и Сережа сделали одинаковое
+# количество журавликов, а Катя сделала в два раза больше журавликов,
+# чем Петя и Сережа вместе?
+
+
+s = int(input("Введите количество сделанных журавликов: "))
+print(int(s / 6), int(s * 2 / 3), int(s / 6))
```

Comment:
Функцию `int` использовать совершенно необязательно. Можно использовать оператор целочисленного отделения. Да, и скобки тут не нужны.
```suggestion
print(s // 6, s * 2 // 3, s // 6)
```